### PR TITLE
added ssl_endpoint_identification_algorithm config option

### PR DESCRIPTION
**Pull request by JackDavidson**
added ssl_endpoint_identification_algorithm config option
matching pull request for output here:
https://github.com/logstash-plugins/logstash-input-kafka/pull/301

### CODE REVIEW

File: lib/logstash/outputs/kafka.rb [hunk 1]
```diff
@@ -140,6 +140,8 @@ class LogStash::Outputs::Kafka < LogStash::Outputs::Base
   config :ssl_keystore_password, :validate => :password
   # The password of the private key in the key store file.
   config :ssl_key_password, :validate => :password
+  # Algorithm to use when verifying host. Set to "" to disable
+  config :ssl_endpoint_identification_algorithm, :validate => :string
```
Comment (robbavey) [inline lib/logstash/outputs/kafka.rb]: Can you set the value explicitly to 'https' to ensure that the behavior remains the same in the (probably somewhat unlikely) event that the Kafka client changes defaults again

**Comment by robbavey** [comment]
@JackDavidson I've included this in a [follow-on commit](https://github.com/logstash-plugins/logstash-output-kafka/pull/213) that fixes an issue which was causing this not to work.

Thanks for the contribution!

**Comment by JackDavidson** [comment]
@robbavey No problem :)

It looks like the change was merged, so should I close this now?

**Comment by JackDavidson** [comment]
Changes merged along another branch.

**Pull request closed without merging**